components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2726,6 +2728,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             case "write_members":
                 return await this.authorizer.check(WriteOrganizationMembers(user.id, orgId), experimentMetadata);
 
+            case "write_settings":
```

Comment:
IMO, `settings` is not the right check for these here. `Settings` refers to the "Allow sharing workspace" etc. For `GitProvider` we should have specific permission

Reply:
For that, you'll need to add them to the schema file as a specific `permission write_git = ...` or something like that

Reply:
Ok! I was unsure, but thought you had settled the other way around. Will do! :+1: